modules/upload/upload.service.js

```diff
@@ -9,8 +9,36 @@ const containerName = 'images';
 const getStream = require('into-stream');
 const Jimp = require("jimp");
 const uniqid = require('uniqid');
+const util = require('util');
 
 class UploadService {
+
+    async uploadResizedImage(size, imageName, image) {
+        const resizedImage = image.resize(size, Jimp.AUTO)
+    
+        const buffer = await new Promise((resolve, reject) => {
+            resizedImage.getBuffer(resizedImage.getMIME(), (err, buffer) => {
+                if(err)
+                    reject(err)
+                resolve(buffer)
+            })
+        })
+        
+        const blobName = imageName;
+        const stream = getStream(buffer);
+        const streamLength = buffer.length;
+
+
+        return await new Promise((resolve, reject) => {
+            blobService.createBlockBlobFromStream(containerName, blobName, stream, streamLength, err => {
+                if (err) {
+                    reject(new Error(err))
```

Comment:
No need to wrap it in `Error`, it is already error instance

Reply:
fixed